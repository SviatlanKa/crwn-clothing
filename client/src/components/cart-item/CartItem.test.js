import React from 'react';
import { shallow } from 'enzyme';
import CartItem from './CartItem';



it('expect render CartItem component', () => {
    const item = {
        imageUrl: 'www.images/blue-hat.jpg',
        price: 100,
        name: 'Blue Hat',
        quantity: 3
    };
    const wrapper = shallow(<CartItem item={item} />);
    expect(wrapper).toMatchSnapshot();
})