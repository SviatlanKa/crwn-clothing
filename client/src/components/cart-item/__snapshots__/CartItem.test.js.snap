// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect render CartItem component 1`] = `
<div
  className="cart-item"
>
  <img
    alt="item"
    src="www.images/blue-hat.jpg"
  />
  <div
    className="item-details"
  >
    <span
      className="name"
    >
      Blue Hat
    </span>
    <span>
      3
       x $
      100
    </span>
  </div>
</div>
`;
